projects/v3/src/app/components/assessment/assessment.component.ts

```diff
@@ -267,6 +270,7 @@ export class AssessmentComponent implements OnChanges {
 
     if (saveInProgress) {
       this.savingMessage = 'Saving...';
+      this.btnDisabled = true;
```

Comment:
We need to disable the button not only when save in progress, but also when submit assessment

Reply:
@shawnm0705 it's already doing by this method `continueToNextTask()`. this is the method call from HTML Button.